stix_shifter/stix_transmission/src/modules/bigfix/bigfix_status_connector.py

```diff
@@ -4,12 +4,17 @@
 import json
 import re
 import sys
+import time
 from .....utils.error_response import ErrorResponder
 import xmltodict
 
+WAIT_PERIOD = 6
```

Comment:
just wondering why `WAIT_PERIOD` is  `6` . is it just an arbitrary number?

Reply:
Yes it is an arbitrary value used for the wait logic.